src/camera.js

```diff
@@ -274,11 +274,11 @@ class IsometricCamera extends Camera {
     }
 
     setupProgram(program) {
-        program.uniform('Matrix4fv', 'u_projection', false, this.projection_matrix);
+        program.uniform('Matrix4fv', 'u_projection', this.projection_matrix);
 
-        program.uniform('3f', 'u_eye', 0, 0, this.viewport_height);
+        program.uniform('3fv', 'u_eye', [0, 0, this.viewport_height]);
```

Comment:
To elaborate: what I mean is, when using `3f`, the individual components of the passed vector will be sent to GL as separate arguments, while with `3fv` the whole array will be sent as one argument. But either way the same array value should cause the same result whether `3f` or `3fv` is specified.